tests/store/createStore.spec.js

```diff
@@ -3,27 +3,23 @@ import {
 } from 'store/createStore'
 
 describe('(Store) createStore', () => {
-  let store
-
-  before(() => {
-    store = createStore()
-  })
-
```

Comment:
why did you favor declaration of store inside each assertions, rather than having them in `beforeEach`?